docs/docs/unreachable.md

```diff
@@ -0,0 +1,90 @@
+return of the lost task, it will be killed immediately after it is seen by the system (assuming it is past expunge time).
+
+When the unreachable strategy is {0,0}, the replacement is near immediate with a replacement commonly running (for the test app) of roughly 3 seconds. The expunge happening around 8-11 seconds. When we include this with the Mesos details provided above for the loss of 1 agent, it means that Marathon will replace a task as soon as it is notified of the unreachable event and it will expunge the original task as soon as it is reachable. If there is time configured for the strategy, then the event cycle that Marathon uses to managing if a task should be replaced or expunged is the task reconciliation cycle. The importance of this is 2 fold: 1) the unreachable strategy time has to expire and 2) the next task reconciliation needs to happen. The task reconciliation is a Marathon system configuration [--reconciliation_interval](https://mesosphere.github.io/marathon/docs/command-line-flags.html). The Marathon defined default is 10 minutes. This means that an unreachable strategy which includes inactiveAfterSeconds = 60, will have a task replaced between 60 seconds and 10 minutes and 60 seconds. The reconciliation interval has the same effect on the expunging of a task.
+
+## DC/OS Unreachable Task Scenarios
+
+The follow scenarios a given for a task going unreachable given a default configuration of a DC/OS cluster. This also assumes that there are resources with required constraints available in the cluster for the replacement task. All marked time references are from the actual loss of an agent from the cluster.
+
+### Strategy {0,0}, Lose 1 node for 4 mins
+
+Marathon is never aware and Mesos reports task.
+
+### Strategy {0,0}, Lose 1 node for 5 mins; Node becomes reachable after 6 mins of original event.
+
+At 5 min mark, Marathon will replace 1 task when TASK_UNREACHABLE status received.
+At 6 min mark, Marathon will kill the task that became reachable.
```

Comment:
that is dependent on `killSelection` which is `YoungestFirst` by default. So in your case, the replacement task will be killed.